### PR TITLE
add to calendar test

### PR DESCRIPTION
**Pull request by jamesgorrie**
screen shot 2018-01-23 at 09 52 50

Only for events that have no other form of booking, and only if there is 1 date.
If this seems to prove popular (via tracking) we could look at implementing it across all dates - but something to explore, not have as part of a 404(ish) test

__TODO__
- [x] create all links
- [x] confirm links works
- [x] add tracking

### CODE REVIEW

**Comment by jamesgorrie** [comment]
Potentially related to:
#2050

**Comment by pollecuttn** [comment]
@jamesgorrie How does this work? Do I click 'Add to calendar' and also 'Outlook'? Why not just 'Outlook'? Sorry if I'm missing something obvious!

**Comment by jamesgorrie** [comment]
@pollecuttn not at all - it's not massively refined as I'd like to see if people use it before refining it.

Add to calendar does nothing. You need to click a link.
So you do just click Outlook.

**Comment by jamesgorrie** [comment]
@wellcometrust/experience-dev looking for :eyes please!

File: events/app/views/pages/event.njk [hunk 1]
```diff
@@ -127,6 +127,44 @@
...
+                  startTime: event.times[0].startDateTime,
+                  endTime: event.times[0].endDateTime,
+                  title: event.title,
+                  description: event.description,
+                  location: 'Wellcome Collection, 183 Euston Rd, Kings Cross, London NW1 2BE'
+                } %}
+                <a data-track-event="{{ {category: 'component', action: 'add-to-calendar:click', label: 'google'} | dump }}"
+                   target="_blank" href="{{ eventCalendarProps | googleCal }}"
+                   class="{{ {s: 1} | spacingClasses({padding: ['right', 'left']}) }}">Google</a>
+                <a data-track-event="{{ {category: 'component', action: 'add-to-calendar:click', label: 'outlook'} | dump }}"
```
Comment (gestchild) [inline events/app/views/pages/event.njk]: label should be google

**Pull request merged**